tempesta_fw/filter.c

```diff
@@ -298,7 +300,8 @@ tfw_filter_stop(void)
 	if (tfw_runstate_is_reconfig())
 		return;
 	if (ip_filter_db) {
-		nf_unregister_hooks(tfw_nf_ops, ARRAY_SIZE(tfw_nf_ops));
+		nf_unregister_net_hooks(&init_net, tfw_nf_ops,
+					ARRAY_SIZE(tfw_nf_ops));
```

Comment:
Why `init_net` and why don't you add the hooks to all network namespaces as other security modules do this using `register_pernet_subsys()`?

Reply:
Corrected.